guidelines/index.html

```diff
@@ -78,1254 +106,1196 @@ <h3>Section status levels</h3>
 			<h2>Guidelines</h2>
 				<details class="summary">
 				<summary>Summary</summary>
-				<p>The following guidelines are being considered for WCAG 3. They are currently a list of topics which we expect to explore more thoroughly in future drafts. The list includes current WCAG 2 guidance and  additional requirements. The list will change in future drafts.</p>
+				<p>The following guidelines are being considered for WCAG 3. They are currently a list of topics which we expect to explore more thoroughly in future drafts. The list includes current WCAG 2 guidance and additional requirements. The list will change in future drafts.</p>
+				<p>Unless otherwise stated, requirements assume the content described is provided both visually and programmatically.</p>
 			</details>
 		
 			
 			<div class="ednote">
-			<p>The individuals and organizations that use WCAG vary widely and include web designers and developers, policy makers, purchasing agents, teachers, and students. To meet the varying needs of this audience, several layers of guidance will be provided including functional categories of disabilities, general guidelines, outcomes that can be tested, a rich collection of methods, resource links, and code samples.</p>
-
-				<p><strong> The following guidelines are an initial list of potential outcomes that the working group will be exploring. The goal is to guide the next phase of work. They should be considered drafts and should not be considered as final content of WCAG 3</strong>.</p>
+			<p>The individuals and organizations that use WCAG vary widely and include web designers and developers, policy makers, purchasing agents, teachers, and students. To meet the varying needs of this audience, several layers of guidance will be provided including guidelines written as outcome statements, requirements that can be tested, assertions, a rich collection of methods, resource links, and code samples.</p>
 
-			<p>Ordinarily, exploratory content includes editor's notes listing concerns and questions for each item.  Because this Guidelines section is very early in the process of working on WCAG 3.0, this editor's note covers the content of the entire section.  Please consider all items in the list as exploratory at this point. It is a list of all possible topics for consideration. Not all items listed will become requirements.</p>
+				<p><strong> The following list is an initial set of potential guidelines and requirements that the working group will be exploring. The goal is to guide the next phase of work. They should be considered drafts and should not be considered as final content of WCAG 3</strong>.</p>
```

Comment:
```suggestion
				<p><strong> The following list is an initial set of potential guidelines and requirements that the Working Group will be exploring. The goal is to guide the next phase of work. They should be considered drafts and should not be considered as final content of WCAG 3</strong>.</p>
```

Reply:
I did a run through of all instances of "working group" to capitalize.